tasks/internal/request-and-upload-signature/tests/mocks.sh

```diff
@@ -16,29 +16,26 @@ function ssh() {
     echo "$ssh_call_count" > "$(workspaces.data.path)/ssh_calls.txt"
 }
 
-function request-signature() {
-  echo Mock request-signature called with: $*
-  echo $* >> $(workspaces.data.path)/mock_request-signature.txt
+function pubtools-sign-msg-container-sign() {
+  >&2 echo "Mock pubtools-sign-msg-container-sign called with: $*"
+  echo "$*" >> "/tmp//mock_pubtools-sign.txt"
+  out=$(sed -n '1p' "$(workspaces.data.path)/mocked_signing_response")
+  sed -i '1d' "$(workspaces.data.path)/mocked_signing_response"
+  status=$(echo "$out" | jq '.signer_result.status' | tr -d \")
+  if [ "${status}" == "ok" ]; then
```

Comment:
What is the point of this? You `echo "$out"` either way

Reply:
fixed